titus-server-master/src/main/java/com/netflix/titus/master/mesos/kubeapiserver/direct/DefaultTaskToPodConverter.java

```diff
@@ -81,6 +84,12 @@
     private static final String ARN_SUFFIX = ":role/";
     private static final Pattern IAM_PROFILE_RE = Pattern.compile(ARN_PREFIX + "(\\d+)" + ARN_SUFFIX + "\\S+");
 
+    /**
+     * Max allowed skew for zone load balancing as a soft constraint. As we do not want to prevent placement in any
+     * case, it must be >= max job size.
+     */
+    private static final int SOFT_MAX_SKEW = 100_000;
```

Comment:
minor: configurable

Reply:
It is set to a very high value so it is always larger than any job.
Would you like to make it configurable, in case we have such big jobs, or to actually have a skew < job size. In the latter case, it should be computed as a % of desired job size.

Reply:
just a minor quality of life change in case we observe skew issues when running this and need to tweak it. Given we are using this with `.whenUnsatisfiable("ScheduleAnyway")` maybe it doesn't matter much